prepare.js

```diff
@@ -1,10 +0,0 @@
-/**
- * This optionally loads husky except in CI environments
- * like Github Actions where the lint-staged pre-commit hook
- * seems to break. CI is always set to true in Github Actions
- * so there's no need to set it manually.
- */
-const isCi = process.env.CI !== undefined;
```

Comment:
Hey @GGKapanadze are you sure this is no longer an issue?

Reply:
During my testing yes, I'll keep an eye on it